docs/quick-start.md

```diff
@@ -210,93 +210,167 @@ kubectl exec -n vm "$VMSINGLE_POD_NAME" -- ls -l  /victoria-metrics-data
 # drwxr-xr-x    3 root     root          4096 Apr 30 12:20 tmp
 ```
 
-### Scraping
-
-#### VMAgent
+While you can push metrics directly as shown above, in most cases you won't need to.
+VictoriaMetrics can discover your applications and scrape their metrics. Let's set scrapping up next.
 
-Now let's deploy [`vmagent`](https://docs.victoriametrics.com/operator/resources/vmagent) resource.
+### Scraping
 
-Create file `vmagent.yaml` 
+The [VMAgent](https://docs.victoriametrics.com/operator/resources/vmagent/) discovers and scrape metrics from your apps.
+It uses special resources like [VMServiceScrape](https://docs.victoriametrics.com/operator/resources/vmservicescrape/) to know where to look.
+Once you create a scrape resource, `VMAgent` picks it up automatically and starts pulling metrics.
 
-```shell
-code vmagent.yaml
-```
-
-with the following content:
-
-```yaml
+Let’s start by creating a `VMAgent` manifest file:
+```bash
```

Comment:
nit: `sh` is the same but shorter shortcode

Reply:
fixed